packages/registry/src/registry/component-metadata.tsx

```diff
@@ -9,9 +9,9 @@ export class ComponentMetadata<Props, State> implements IComponentMetadata<Props
   public nonA11yCompliant: boolean = false;
   public nonEventListenerTestCompliant: boolean = false;
   public nonHydrationTestCompliant: boolean = false;
-  public path: string = '';
-  public exportName: string = '';
-  public baseStylePath: string = '';
+  public exportInfo: IExportInfo = {path: '', exportName: ''};
```

Comment:
I'd prefer `IExportInfo | null`. Seems like `if (metadata.exportInfo) { /* use it */ }` is a bug waiting to happen.

Reply:
Great, will change